system-int-test/test/cmr/system_int_test/ingest/granule_ingest_test.clj

```diff
@@ -73,6 +75,19 @@
                       (:concept-id coll1) (:concept-id coll2))]
              errors)))))
 
+;; Verify that granule indexing is not being changed too quickly.
```

Comment:
I see you also rely on the printed out new index list to update the existing one. so you will need to keep that (is (= allowed-index actual-index)) thing. then maybe you could add what I suggested and when granule index changed, actual-string="granule index changed wait till next sprint to change search using the index. Also update the allowed-index in <file> using the actual index printed out to make the test pass"  what do you think?